### ISSUE TITLE
Use case ideas and discussion.

### ISSUE THREAD

**Issue by vishalvibes** [houseofgeeks/CP-Dictionary#157]
Can anyone suggest ideas for using this dictionary for directly benefiting coders?

**Comment by sanskarseth** [comment]
Links of questions using the concept can be attached to every topic.

**Comment by vishalvibes** [comment]
@sanskarseth yes this can be done...we can make a .md file for every algorithm with related blogs and good resources
